sources/api/apiclient/src/main.rs

```diff
@@ -1064,7 +1070,8 @@ async fn run() -> Result<()> {
                 .context(error::EphemeralStorageSnafu)?;
             }
             EphemeralStorageSubcommand::Bind(bind_args) => {
-                ephemeral_storage::bind(&args.socket_path, bind_args.targets)
+                let targets = bind_args.targets;
+                ephemeral_storage::bind(&args.socket_path, targets)
```

Comment:
Nit: can drop the local variable `target` here? Essentially keep the original code.